eng/testing/tests.mobile.targets

```diff
@@ -82,12 +85,16 @@
       <AndroidAbi Condition="'$(TargetArchitecture)' == 'x86'">x86</AndroidAbi>
 
       <MainLibraryFileName Condition="'$(MainLibraryFileName)' == ''">AndroidTestRunner.dll</MainLibraryFileName>
+      <RuntimeConfigFilePath>$(PublishDir)$(AssemblyName).runtimeconfig.json</RuntimeConfigFilePath>
+      <OutputFilePath>$(PublishDir)output.pefile</OutputFilePath>
     </PropertyGroup>
     <ItemGroup Condition="'$(RunAOTCompilation)' == 'true'">
       <AotInputAssemblies Include="$(PublishDir)\*.dll">
         <AotArguments>@(MonoAOTCompilerDefaultAotArguments, ';')</AotArguments>
         <ProcessArguments>@(MonoAOTCompilerDefaultProcessArguments, ';')</ProcessArguments>
       </AotInputAssemblies>
+      <ReservedProperties Include="RUNTIME_IDENTIFIER"/>
+      <ReservedProperties Include="APP_CONTEXT_BASE_DIRECTORY"/>
```

Comment:
again `ReservedProperties` is a very generic global name.  How aout `RuntimeConfigReservedProperties`?

Reply:
Fixed. Thanks!